data/mods/ssb/abilities.ts

```diff
@@ -349,6 +349,22 @@ export const BattleAbilities: {[k: string]: ModdedAbilityData} = {
 		},
 		name: "Venomize",
 	},
+	
+	// Jordy
+	archeopssrage: {
+		desc: "Sets sandstorm on switch-in and does not take recoil damage.",
```

Comment:
You can literally just take the descriptions from both abilities and copypaste them here.